exporter/elasticsearchexporter/internal/objmodel/objmodel_test.go

```diff
@@ -372,13 +372,15 @@ func TestDocument_Serialize_Dedot(t *testing.T) {
 
 func TestValue_Serialize(t *testing.T) {
 	tests := map[string]struct {
-		value Value
-		want  string
+		value   Value
+		want    string
+		wantErr error
```

Comment:
wantErr is unused. Do you plan to write a test that walks that code path?

Reply:
Ah right, I added previously, when we returned errors. But we don't anymore.